### ISSUE TITLE
RemovedInDjango19Warning

### ISSUE THREAD

**Issue by socketubs** [BertrandBordage/django-cachalot#12]
When running django-cachalot with Django 1.8 `./manage.py test`:

```
Creating test database for alias 'default'...
/home/socketubs/.virtualenvs/bounty-o-matic/lib/python3.4/site-packages/cachalot/cache.py:34: RemovedInDjango19Warning: 'get_cache' is deprecated in favor of 'caches'.
    return get_django_cache(cache_alias)
```

**Comment by socketubs** [comment]
@BertrandBordage No problem, I don't use Django 1.8 for now, this is just to notify you about it :smile:

**Comment by BertrandBordage** [comment]
Fixed with 16e14c92192f22269dcbedb73a48be8e7c1204ad.

**Comment by socketubs** [comment]
Perfect! :smile:
